tests/systemtests/system.go

```diff
@@ -134,11 +134,6 @@ func (s *SystemUnderTest) SetupChain() {
 	if err != nil {
 		panic(fmt.Sprintf("failed to set block max gas: %s", err))
 	}
-	// Short period for gov
-	genesisBz, err = sjson.SetRawBytes(genesisBz, "app_state.gov.params.voting_period", []byte(fmt.Sprintf(`"%s"`, "8s")))
-	if err != nil {
-		panic(fmt.Sprintf("failed to set regular voting period: %s", err))
-	}
 	// update expedited voting period to avoid validation error
 	genesisBz, err = sjson.SetRawBytes(genesisBz, "app_state.gov.params.expedited_voting_period", []byte(fmt.Sprintf(`"%s"`, "7s")))
```

Comment:
@hieuvubk Can you move this too to gov_test.go? This was added due to voting_period change earlier.